src/server/.eslintrc.js

```diff
@@ -1,18 +1,16 @@
 module.exports = {
-	env: {
-		commonjs: true,
-		es6: true,
-		node: true
+	'env': {
+	  'es6': true
 	},
-	extends: 'standard',
-	globals: {
-		Atomics: 'readonly',
-		SharedArrayBuffer: 'readonly'
+	'extends': 'standard',
```

Comment:
Fix problem with eslint
include Airbnb, please

Reply:
airbnb linter added to server